### PR TITLE
Added Tutorial of LOJ 1017 - BRUSH(III) (en)

### PR DESCRIPTION
**Pull request by ahrifatarafat**
added English tutorial and cpp code and English resources.

### CODE REVIEW

File: 1017/en.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+ -  4.[Geekforgeeks : Binary Search](https://www.geeksforgeeks.org/binary-search/)
+___
+### Solution:
+Basically, we need only y axis point.  
+<img width="1200" alt="initial" src="https://github.com/ahrifatarafat/Lightoj-Tutorial/blob/main/LOJ_1017/LOJ_1017.JPG">
+Here we see (0,0); (0,5); (20,5); (100,8); the four points then which way we solve we don’t need x axis points.
+Then sort the y axis points and then we calculate in every moves how many dusts we can clean.
+And also observe that if we start from non-overlapping points then we can clean most of them.
+We first stored if we clean this dust then how many of them doesn’t overlapped before we can clean. And also how many of them we can take from front for clean. Then We can check in every moves how many dust we can clean.
+We run a for loop from 0 to k for every moves and iterate over y axis point and calculate the maximum number of dust we can clean on index j of y axis after i moves and stored in dp[j].
```
Comment (rebornplusplus) [inline 1017/en.md:22]: Ambiguous. Perhaps due to grammatical problems?
Also, pleases clearly state what your dp function does, states, transitions and base cases.

File: 1017/en.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+1 57
+
+1 72
+
+1 83
+
+1 94
+
+1 97
```
Comment (rebornplusplus) [inline 1017/en.md:50]: Enclose this in a code block by adding ` ``` ` before and after.

File: 1017/en.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+ using namespace std;
+ void solve()
+ {
+ 	int n,w,k,x;
+ 	scanf("%d%d%d",&n,&w,&k);
+  vector<int>a(n+1);
+ 	int dp[n+5];
+ 	vector<int>prev(n+1),mx(n+1, 0),poss(n+1);
+ 	memset(dp, 0, sizeof dp);
+ 	for(int i = 0; i < n; i++)scanf("%d%d" ,&x ,&a[i]);
```
Comment (rebornplusplus) [inline 1017/en.md:83]: Indent code.

**Review by rebornplusplus — CHANGES_REQUESTED** [review]
Hello. Thank you for your efforts. Please see to the changes requested.

File: 1017/en.md (hunk 3 above)
Comment (ahrifatarafat) [reply 1017/en.md:83]: I am now working on this. Thanks for Reply.

**Comment by ahrifatarafat** [comment]
I updated my tutorial. Can you just check it?

**Review by rebornplusplus — CHANGES_REQUESTED** [review]
You have `1017/en.md` and `en.md`. Please erase the latter file (the one in `/`). I went through `1017/en.md` and the solution seems good. Although I might add some lines in a separate PR to make it better some time in the future. But your indentation is still messed up. Do fix that.

Changes to make:
* Remove `en.md` from `/`.
* Indent code properly.

**Review by rebornplusplus — APPROVED** [review]
Looks great! :100:
Thank you for your contribution.

**Pull request merged**